backend/internal/controllers/middleware/requireAuth.go

```diff
@@ -0,0 +1,41 @@
+package middleware
+
+import (
+	"context"
+	"net/http"
+	"strings"
+
+	"github.com/TritonSE/words-alive/internal/auth"
+)
+
+// HTTP middleware setting a value on the request context
```

Comment:
This comment is so vague, it should definitely mention that it checks the Authorization header on each request against the Authenticator it is given.